src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -6,55 +6,99 @@
 
 public class Range<T> implements Set<T> {
```

Comment:
You have to implement Set (which contains unique elements).

Reply:
What does it mean?

Reply:
that is to be done in this way?                                                                                                     public class Range<T>  implements Set<T>  {

    private T arrayElement[];
    int size;